include/zephyr/bluetooth/gap.h

```diff
@@ -502,15 +839,15 @@ enum {
 
 /** @brief Peripheral sleep clock accuracy (SCA) in ppm (parts per million) */
 enum {
-	BT_GAP_SCA_UNKNOWN = 0,
-	BT_GAP_SCA_251_500 = 0,
-	BT_GAP_SCA_151_250 = 1,
-	BT_GAP_SCA_101_150 = 2,
-	BT_GAP_SCA_76_100 = 3,
-	BT_GAP_SCA_51_75 = 4,
-	BT_GAP_SCA_31_50 = 5,
-	BT_GAP_SCA_21_30 = 6,
-	BT_GAP_SCA_0_20 = 7,
+	BT_GAP_SCA_UNKNOWN = 0,   /**< Unknown */
```

Comment:
Ditto w.r.t. same line versus the line before it

Reply:
Happy to change everything to "line before" construct, if you think this is more consistent that way? I like how the `/**<` construct keeps things more concise though.

Reply:
We use `/**<` plenty of places in Zephyr that it's fine :D I was mostly curious as why you've opted for both ways in this PR :D